### DISCUSSION TITLE
Velocity model from h5 to SEGY

### DISCUSSION THREAD

**Discussion by NikoYura** [slimgroup/JUDI.jl#229] [Q&A] [answered]
Hello again!

I did not find any possibility of writing the velocity model from h5 format to SEGY using SEGYIO. I am sure that it should be quite easy task. Thank you for any help!

**Comment by mloubout** [comment, marked as answer]
Hi

While we do support some standard SegY for data through our OOC geometries, model input/output is left to the choice of the user as there are too many possibilities. 
Is there any reason you need such a conversion? HDF5 should be fine for velocity. But if you need the velocity in SegY forwat you should be able to do it following SegyIO's Wiki easily

https://github.com/slimgroup/SegyIO.jl/wiki/Writing

**Reply by NikoYura** [reply]
Thank you @mloubout ! I think I had problems with headers and tags, since SeiSee couldn't read them automatically from the generated file by SeisIO. But I've managed to set them manually.
